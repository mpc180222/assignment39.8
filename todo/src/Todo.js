import React from 'react';


const Todo = ({todo, removeTodo, id}) => {

return(

    <div>
    <h2>{todo}</h2>
    <button data-testid={`remove-${todo}`} id = {id} onClick = {removeTodo}>X</button>
    </div>
)



}





export default Todo;