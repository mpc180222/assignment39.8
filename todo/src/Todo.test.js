import React from 'react';
import { render, screen } from '@testing-library/react';
import Todo from './Todo';

it('renders without crashing', () => {
  render(<Todo />);
  
});

it("should match snapshot", () => {
  const { asFragment } = render(<Todo />);
  expect(asFragment()).toMatchSnapshot();

})