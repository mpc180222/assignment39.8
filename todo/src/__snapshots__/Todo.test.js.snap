// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <h2 />
    <button
      data-testid="remove-undefined"
    >
      X
    </button>
  </div>
</DocumentFragment>
`;
